core/app/models/spree/store.rb

```diff
@@ -17,6 +17,8 @@ class Store < Spree::Base
 
     has_one_attached :logo
 
+    validates :logo, content_type: [:png, :jpg, :jpeg, :gif]
```

Comment:
@MatthewKennedy what about SVG?

Reply:
I think I tried .svg as a file type, but the resize causes an error front end and back.

Reply:
@damianlegawiec https://github.com/spree/spree/blob/caeb26d6761851e7b032832f48e1bee39418094c/core/app/helpers/spree/base_helper.rb#L44 

The resize in this method causes an error on svg file types on the front end. I'll just fire up this branch and test that to make sure.

Reply:
@MatthewKennedy you're correct - we should only use resize if the file is an image

Reply:
@damianlegawiec if you take off the resize on the backend, and then upload an .svg  it won't display, I suspect this may be because it is used within img tags, or maybe it's something deeper than that, I haven't looked just yet.

<img width="358" alt="Screenshot 2020-08-07 at 13 56 39" src="https://user-images.githubusercontent.com/1240766/89647680-d3a97600-d8b5-11ea-95d6-548ce0f0483b.png">

I suspect that using an .svg as store logo would need to be handled completely differently to you're regular images.